lib/model/rwfolder.go

```diff
@@ -666,6 +674,77 @@ func (f *rwFolder) handleDir(file protocol.FileInfo) {
 	}
 }
 
+// handleSymlink creates or updates the given symlink
+func (f *rwFolder) handleSymlink(file protocol.FileInfo) {
+	var err error
+	events.Default.Log(events.ItemStarted, map[string]string{
+		"folder": f.folderID,
+		"item":   file.Name,
+		"type":   "symlink",
+		"action": "update",
+	})
+
+	defer func() {
+		events.Default.Log(events.ItemFinished, map[string]interface{}{
+			"folder": f.folderID,
+			"item":   file.Name,
+			"error":  events.Error(err),
+			"type":   "symlink",
+			"action": "update",
+		})
+	}()
+
+	realName, err := rootedJoinedPath(f.dir, file.Name)
```

Comment:
Oh. Yes, that actually works, although it's quite non-obvious and not really something I'd recommend, even though it's probably me who wrote it... Since err is already declared in the same scope, the `:=` does nothing on it.